src/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2024 Renesas Electronics Corporation.
+# Copyright (C) 2024 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET aos_iamanager)
+
+# ######################################################################################################################
+# Defines
+# ######################################################################################################################
+
+# ######################################################################################################################
+# Includes
+# ######################################################################################################################
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+# ######################################################################################################################
+# Sources
+# ######################################################################################################################
+
+set(SOURCES main.cpp)
+
+# ######################################################################################################################
+# Target
+# ######################################################################################################################
+
+add_executable(${TARGET} ${SOURCES})
+
+# ######################################################################################################################
+# Labraries
+# ######################################################################################################################
+
+target_link_libraries(${TARGET} PUBLIC app logger)
+
+# ######################################################################################################################
+# Install
+# ######################################################################################################################
+
+install(TARGETS ${TARGET} RUNTIME DESTINATION bin)
+
+# ######################################################################################################################
+# Modules
+# ######################################################################################################################
+
+add_subdirectory(app)
+add_subdirectory(logger)
```

Comment:
I think the ordering is not good:
modules should be imported before they are linked

Reply:
Doesn't matter actually, but I'm ok to move them to the top.